tests/test_sliding_patch_wsi_dataset.py

```diff
@@ -226,11 +222,11 @@ def setUpClass(cls):
         cls.backend = "cucim"
 
 
-@skipUnless(has_osl, "Requires openslide")
-class TestSlidingPatchWSIDatasetOpenSlide(SlidingPatchWSIDatasetTests.Tests):
-    @classmethod
-    def setUpClass(cls):
-        cls.backend = "openslide"
+# @skipUnless(has_osl, "Requires openslide")
```

Comment:
this change seems to be unrelated to #4179?

Reply:
That's commented out by mistake! Thanks for pointing it out.